### DISCUSSION TITLE
FATA[0000] rpc error: code = PermissionDenied desc = permission denied: applications, create

### DISCUSSION THREAD

**Discussion by jesusroval** [argoproj/argo-cd#7347] [General]
hello team,
please help me:
I have issue with operator argocd with openshift 4.8. when I want to create new app
FATA[0000] rpc error: code = PermissionDenied desc = permission denied: applications, create ... 

thank you team

**Comment by alexmt** [comment]
Hello @jesusroval ,

According to the error message, you are using Argo CD CLI to create an application (`argocd app create`), correct?
The `PermissionDenied` means that your Argo CD user doesn't have permission to create applications.

You can get information about your user using `argocd account get-user-info` command. The command should return list of groups your user belongs to. In order to create applications the user must be a member of a group that have permissions to create applications. More information about permissions is available here: https://argo-cd.readthedocs.io/en/stable/operator-manual/rbac/
